components/docs/book/TitleFolderNaming.vue

```diff
@@ -49,13 +53,21 @@
         <img src="/folder.svg" class="h-6" />
         <p class="pl-2">1 - Wizards First Rule</p>
       </div>
+      <div class="flex py-1">
+        <img src="/folder.svg" class="h-6" />
+        <p class="pl-2">1. Wizards First Rule</p>
+      </div>
     </div>
 
     <ul class="list-outside md:list-inside px-6 md:px-0 list-disc my-4 text-sm md:text-base">
-      <li class="py-1"><strong class="underline">Subtitle: </strong>Parsing out subtitles into a separate field is optional and must be enabled in settings. Subtitle must be separated by " - ".</li>
-      <li class="py-1"><strong class="underline">Volume Number: </strong>The volume number can be the first number in the folder name separated by " - " (see last example) OR anywhere in the folder name if "Vol", "Book", or "Volume" is before the number. Case insensitive & decimals supported.</li>
-      <li class="py-1"><strong class="underline">Publish Year: </strong>The publish year must be the first part of the name OR directly after a volume number, and separated by " - " on both sides.</li>
-      <li class="py-1"><strong class="underline">Narrator: </strong>The Narrator must be at the end of the folder name in {curly braces}.</li>
+      <li class="py-1"><strong><u>Subtitle</u>:</strong> Parsing out subtitles into a separate field is optional and must be enabled in settings. Subtitle must be separated by " - ".</li>
+      <li class="py-1"><strong><u>Volume Number</u>:</strong> Case insensitive & decimals supported.</li>
+        <ul>
+          <li style="padding-left: 3em">The volume number can be placed anywhere in the folder name.</li>
+          <li style="padding-left: 3em">It must be followed by " - " or ". "</li>
+          <li style="padding-left: 3em">If it is not at the beginning of the folder name, it must be preceded by " - " and "Vol" "Vol." "Volume" or "Book"</li>
```

Comment:
The information here is correct. I'm not convinced this is the best formatting choice, but it's not terrible. I'm super open to other ideas about that.

Reply:
Looks good to me